src/views/confirmation/confirmation.view.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC, useEffect, useState } from "react";
+
+import { ReactComponent as ArrowRightIcon } from "src/assets/icons/arrow-right.svg";
+import useConfirmationStyles from "src/views/confirmation/confirmation.styles";
+import Header from "src/views/shared/header/header.view";
+import { useTransactionContext } from "src/contexts/transaction.context";
+import Card from "src/views/shared/card/card.view";
+import TokenIcon from "src/views/shared/token-icon/token-icon.view";
+import Typography from "src/views/shared/typography/typography.view";
+import { useNavigate } from "react-router-dom";
+import routes from "src/routes";
+import { tokenAmountToNumber, convertTokenAmountToFiat } from "src/utils/amounts";
+import Button from "src/views/shared/button/button.view";
+import Error from "src/views/shared/error/error.view";
+
+const Confirmation: FC = () => {
+  const classes = useConfirmationStyles();
+  const [isDisabled, setIsDisabled] = useState(false);
+  const navigate = useNavigate();
+  const { transaction } = useTransactionContext();
+
+  useEffect(() => {
+    //TODO Check network connected
+  }, [setIsDisabled]);
+
+  useEffect(() => {
+    if (!transaction) {
+      navigate(routes.home.path);
+    }
+  }, [navigate, transaction]);
+
+  if (!transaction) {
+    return null;
+  }
+
+  const onClick = () => null;
```

Comment:
You can write this code here to do a bridge transaction as well by the moment so I can remove it. from my `Home` view:

```
const { token, amount, to } = transaction;

bridge(token.address, amount, to.chainId, account.data)
        .then(console.log)
        .catch(console.error);
```